### DISCUSSION TITLE
Bal is corrupt in 5.0.0?

### DISCUSSION THREAD

**Discussion by bartwalczak** [wixtoolset/issues#8175] [Questions]
```
> dotnet tool install --global wix
Tool 'wix' was reinstalled with the stable version (version '5.0.0').

> wix extension add WixToolset.Bal.wixext
> wix extension list
WixToolset.Bal.wixext 5.0.0 (damaged)
```

Any way to resolve it or do I need to go back to 4.0.4?

**Comment by barnson** [comment]
https://wixtoolset.org/docs/fivefour/oopbas/

**Reply by l0rd** [reply]
It may be hard to find the answer to the original question from the above link content (at least it was for me).  The important part is:

> WixToolset.Bal.wixext - renamed to WixToolset.BootstrapperApplications.wixext but also kept WixToolset.Bal.wixext for backwards compatibility with PackageReference in MSBuild projects. Note: When using wix.exe, you need to use the new name WixToolset.BootstrapperApplications.wixext

And that worked:

```pwsh
> wix extension add -g WixToolset.BootstrapperApplications.wixext
> wix extension list -g
WixToolset.BootstrapperApplications.wixext 5.0.
```

The [Wix Extensions documentation page](https://wixtoolset.org/docs/tools/wixext/) is still referring to the wrong extension name `WixToolset.Bal.wixext` though. [I have submitted a PR to address that](https://github.com/wixtoolset/web/pull/286).
